demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -0,0 +1,216 @@
+"""
+This script is used to create a release notes template
+"""
+
+import errno
+import json
+import os
+import sys
+
+from demisto_sdk.commands.common.constants import PACKS_PACK_META_FILE_NAME
+from demisto_sdk.commands.common.hook_validations.structure import \
+    StructureValidator
+from demisto_sdk.commands.common.tools import (LOG_COLORS, get_json,
+                                               pack_name_to_path, print_color,
+                                               print_error, print_warning)
+
+
+class UpdateRN:
+    def __init__(self, pack: str, update_type: str, pack_files: set, pre_release: bool = False):
+
+        self.pack = pack
+        self.update_type = update_type
+        self.pack_meta_file = PACKS_PACK_META_FILE_NAME
+        self.pack_path = pack_name_to_path(self.pack)
+        self.metadata_path = os.path.join(self.pack_path, 'pack_metadata.json')
+        self.pack_files = pack_files
+        self.pre_release = pre_release
+
+    DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+    def execute_update(self):
+        try:
+            new_version = self.bump_version_number(self.pre_release)
+        except ValueError as e:
+            print_error(e)
+            sys.exit(0)
```

Comment:
not sure. when getting an error should we exit(0)? @bakatzir WDYT?

Reply:
The convention is `return 1` for errornous execution, `return 0` for a successful one.